README.md

```diff
@@ -240,10 +239,9 @@ When a PostgreSQL Operator general availability (GA) release occurs, the contain
 
 - [Crunchy Data Customer Portal](https://access.crunchydata.com/)
 - [Crunchy Data Developer Portal](https://www.crunchydata.com/developers)
-- [DockerHub](https://hub.docker.com/u/crunchydata)
 
 The image rollout can occur over the course of several days.
 
-To stay up-to-date on when releases are made available in the [Crunchy Data Developer Portal](https://www.crunchydata.com/developers), please sign up for the [Crunchy Data Developer Program Newsletter](https://www.crunchydata.com/developers/newsletter). You can also [join the PGO project community mailing list](https://groups.google.com/a/crunchydata.com/forum/#!forum/postgres-operator/join)
+To stay up-to-date on when releases are made available in the [Crunchy Data Developer Portal](https://www.crunchydata.com/developers), please sign up for the [Crunchy Data Developer Program Newsletter](https://www.crunchydata.com/developers#email). You can also [join the PGO project community mailing list](https://groups.google.com/a/crunchydata.com/forum/#!forum/postgres-operator/join)
```

Comment:
There's no separate page for newsletter, but there's a tag on the `email` section to sign up for the newsletter at the bottom of the developers page